.github/workflows/python-package-conda.yml

```diff
@@ -1,4 +1,4 @@
-name: Python Package using Conda
+name: Build
```

Comment:
"Build and Test" would be more accurate

Reply:
As this will also change the badge on the readme, I would keep it as simple as possible. Hence just Build. But please add it later if you prefer it.